doc/operation-and-maintenance/Cluster-configuration-and-node-management.md

```diff
@@ -53,71 +53,51 @@ Make sure that the range you set provides enough ports for all the nodes in the
 
-## Load Balancing
-
-### Elastic Load Balancer (ELB)
-
-When using ELB pleas be advised that many of our users reported it to prematurely drop connections, especially when using BOSH.
-
-### DNS-based load balancing
-
-Load balancing can be performed on a DNS level. 
-A DNS response can have a number of IP addresses that can be returned to the client side in a random order. 
-
-On the AWS stack this type of balancing is provided by Route53. 
-The description of their service can be found at:
-
-http://docs.aws.amazon.com/Route53/latest/DeveloperGuide/WeightedResourceRecordSets.html
-
-### Software load balancer
-
-A good example of load balancing on the application layer are HAProxy and Nginx. 
-
-### Other
-
-The approaches described above can be mixed - we can use DNS load balancing to pick a software load balancer which will select one of the nodes.
-
 ## Connecting nodes
 
 Checklist:
 
-- working directory `rel/mongooseim`
+- working directory `rel/mongooseim` (root of MongooseIM release or installation)
 - the same cookie across all nodes (`vm.args` `-setcookie` parameter)
 - each node should be able to ping other nodes using its sname
    (ex. `net_adm:ping('mongoose@localhost')`)
-- no mnesia files - no `Mnesia.mongooseim@[host name]` directory in cwd on new node's release
 
 ### Initial node
 
 There is no action required on the initial node.
 
 Just start MongooseIM using `mongooseim start` or `mongooseim live`.
 
-### New node
+### New node - joining cluster
 
-#### MongooseIM 1.7.0 and newer
-
-Cluster management commands changed a bit since MongooseIM 1.7.0. 
-Now we call them from a running node that is going to leave or join the cluster.
 
 ```bash
+mongooseimctl start
+mongooseimctl started #waits until MongooseIM starts
 mongooseimctl join_cluster ClusterMember
 ```
 
-`ClusterMember` is the name of a running node set in `vm.args` file, for example `mongooseim@localhost`. 
-This nodehas to be part of the cluster we'd like to join.
+`ClusterMember` is the name of a running node set in `vm.args` file, for example `mongooseim@localhost`.
+This node has to be part of the cluster we'd like to join.
+
+First MongooseIM will display a warning and a question if the operation should proceed:
 
-The successful output from the above command starts with `You have successfully joined the node to the cluster`.
+```
+Warning. This will drop all current connections and will discard all persistent data from Mnesia. Do you want to continue? (yes/no)
+```
 
-#### MongooseIM before 1.7.0
+If you type `yes` MongooseIM will start joining the cluster.
+Successful output may look like the following:
 
-For an older `MongooseIM` please see the previous versions of the documentation.
+```
+You have successfully joined the node mongooseim2@localhost to the cluster with node member mongooseim@localhost
+```
 
-### Remove node
+In order to skip the question you can use `-f` which will perform the action without any warning and interaction.
```

Comment:
In order to skip the question you can use `-f` which will perform the action without any warning or interaction?

Reply:
How about:

In order to skip the question you can add option `-f` which will perform the action without displaying the warning and waiting for confirmation

Does it sound better @goddammit ?

Reply:
In order to skip the question you can add option -f which will perform the action without displaying the warning and waiting for the confirmation.

:)